bciers/apps/reporting/src/app/components/complianceSummary/ComplianceSummaryPage.tsx

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import { actionHandler } from "@bciers/actions";
+import ComplianceSummaryForm from "./ComplianceSummaryForm";
+import { tasklistData } from "./TaskListElements";
+import { HasReportVersion } from "@reporting/src/app/utils/defaultPageFactoryTypes";
+import { getRegistrationPurpose } from "@reporting/src/app/utils/getRegistrationPurpose";
+import { getAttributableEmissions } from "@reporting/src/app/utils/getAttributableEmissions";
+import {
+  RegistrationPurposes,
+  regulatedOperationPurposes,
+} from "@/registration/app/components/operations/registration/enums";
+
+const getComplianceData = async (versionId: number) => {
+  return actionHandler(
+    `reporting/report-version/${versionId}/compliance-data`,
+    "GET",
+    `reporting/report-version/${versionId}/compliance-data`,
+  );
+};
+export default async function ComplianceSummaryPage({
+  version_id,
+}: HasReportVersion) {
+  const complianceData = await getComplianceData(version_id);
+  //🔍 Check if reports need verification step...
+  let needsVerification = false;
+  //🔍 Check if registration purpose is OBPS Regulated Operation, Opt-in or New Entrants
+  const registrationPurpose = (await getRegistrationPurpose(version_id))
+    ?.registration_purpose;
+  needsVerification = regulatedOperationPurposes.includes(
+    registrationPurpose as RegistrationPurposes,
+  );
+  if (
+    needsVerification === false &&
+    registrationPurpose === RegistrationPurposes.REPORTING_OPERATION
+  ) {
+    //🔍 Check if the registration purpose is Reporting Operation AND their total emissions attributable for reporting threshold is = or > than 25,000 TCo2
+    const attributableEmissionThreshold = 25000000;
+    const attributableEmissions = await getAttributableEmissions(version_id);
+    needsVerification = attributableEmissions >= attributableEmissionThreshold;
+  }
```

Comment:
We should do this on the API side, otherwise technically an individual with malicious intentions might bypass it on the client.
It'll also allow us to refactor that logic for other forms under a service of some sorts, like a follow up to https://github.com/bcgov/cas-reporting/issues/445 and https://github.com/bcgov/cas-reporting/issues/450

Reply:
@pbastia 

For your consideration, as this component is a **server side component**; perhaps, it is as safe from mal-intent as an API ?